components/layout-container/layout-container.html

```diff
@@ -0,0 +1,12 @@
+<style>
+    * {
+        box-sizing: border-box;
```

Comment:
Are we expecting border styling on this?

Reply:
The .html of the component has now been removed as it is not required anymore.